src/common/snippets/include/snippets/pass/insert_movebroadcast.hpp

```diff
@@ -22,6 +22,17 @@ class InsertMoveBroadcast: public ngraph::pass::MatcherPass {
     InsertMoveBroadcast();
 };
 
+/**
+ * @interface BroadcastToMoveBroadcast
+ * @brief Inserts explicit MoveBroadcast instruction if broadcasting by most varying dimension is needed instead of Broadcast.
+ *        Otherwise the pass removes Broadcast operation.
```

Comment:
Thank you for you comment!
I see your concern. So Snippets will tokenize `Broadcast` nodes which don't impact on the result shape. You can take a look on the [code](https://github.com/a-sidorova/openvino/blob/4166c9ae8c3d308c75990256b6979600690ad818/src/common/snippets/src/pass/mha_tokenization.cpp#L48) in MHA Tokenization. At the moment it's enough to support `Broadcast` op in distilbert models. And it's our sub-task. If we want to support `Broadcast` in general, we should investigate it better. I think we don't have enough time for that at the moment. Also it's not our target.
As for your suggestion, It'd be great if we analyze and investigate them in bounds of shape inference in Snippets (may be another IR for shape propagation)

Reply:
Ok, I see your point. Could you please create a ticket for a general-pipeline `Broadcast` support? It looks like we need to document these concerns (and they are currently addressed in the code), because it's likely that we won't remember any of these in a month 😅

Reply:
You're right! The ticket 100040 has been created